client/app/bundles/shared/components/Switch.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from 'react';
+import css from './Switch.scss';
+
+type Props = {
+  id?: string
+};
+
+type State = {
+  checked: boolean
+};
+
+export default class Switch extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { checked: this.props.checked || false };
```

Comment:
You could do `!!this.props.checked` instead

Reply:
In line 16, why is it 'this.props.checked' and not 'this.state.checked'?

Reply:
Ooh, you're right it should be `!!this.state.checked`